routes/index.js

```diff
@@ -567,8 +567,7 @@ router.route('/resetpw')
         if(req.body.password !== req.body.confirmpw) {
             return res.redirect('/resetpw?e=unmatched');
         } else {
-            var hashedPW = hash(password);
-
+            var hashedPW = hash(req.body.password);
```

Comment:
@tashrifbillah   🐛 Fix typo in resetpw route

Reply:
Is this why reset password was not working so far?

Reply:
Yes, at least since my update (I introduced that bug).